.github/workflows/pull_request.yml

```diff
@@ -52,3 +54,22 @@ jobs:
           gcloud auth configure-docker ${{ secrets.GAR_LOCATION }}
           echo "Pushing to GAR with tag $TAG"
           docker push ${{ secrets.GAR_LOCATION }}/${{ secrets.GAR_PROJECT_ID }}/docker-images/eq-questionnaire-launcher:$TAG
+  lint:
```

Comment:
The lint step should come before the other steps, so that we aren't pushing images that need to be fixed

Reply:
Updated 👍